test/Engine/FFITarget/DummyGeometry.cs

```diff
@@ -115,6 +116,13 @@ public static DummyLine ByStartPointEndPoint(DummyPoint a, DummyPoint b)
             return ln;
         }
 
+        public static DummyLine ByStartPointEndPointWithDefault(
+            [DefaultArgumentAttribute("DummyPoint.ByCoordinates(0,0,0)")] DummyPoint p1,
```

Comment:
Does it make sense to have a negative test where the DSExpression is syntactically and semantically wrong?

wrong syntax
DummyPoint.ByCoordinates0,0,0)

wrong sematnic (ByCoords does not exist)
DummyPoint.ByCoords(0,0,0)